package no.nav.syfo.exception

class FailedSendingToEsyfovarselException(cause: Throwable) : Exception(cause)
